space/Dockerfile.space

```diff
@@ -81,3 +81,5 @@ ENV NEXT_TELEMETRY_DISABLED=1
 ENV TURBO_TELEMETRY_DISABLED=1
 
 EXPOSE 3000
+
+CMD ["node", "space/server.js"]
```

Comment:
@sriramveeraghanta I left these CMD in so if someone wanted to run the docker containers on their own, they could without having to fumble around for a command.